features/contact/requesting_contact_validation.feature

```diff
@@ -34,7 +34,7 @@ Feature: Requesting contact validation
         And I try to send it
         Then I should be notified that the email is invalid
 
-    @ui
+    @ui @no-api
```

Comment:
channel with missing contact email is a configuration error. There is no way to validate it in a simple way and there is also no reason to do it

Reply:
What will be the current response in API in such a case?

Reply:
you will get 500 internal error